src/MoBi.Presentation/DTO/ObjectBaseDTO.cs

```diff
@@ -132,6 +132,16 @@ public ObservedDataViewItem(DataRepository repository)
       }
    }
 
+   public class SimulationSettingsViewItem : IViewItem
```

Comment:
I don't quite get why we need the simulationSettingsViewItem? Is this required for the context menu to have an IViewIem ?Because it looks like just a wrapper over SimulationSettings

Reply:
Yes, I think context menu needs an IViewItem. I think we have lots of these ObservedDataViewItem, SimulationViewItem and others. I assume that's why.